app/src/main/java/com/safetyheads/akademiaandroida/PasswordValidator.kt

```diff
@@ -0,0 +1,24 @@
+package com.safetyheads.akademiaandroida
+
+import android.text.InputFilter
+import android.text.Spanned
+import android.widget.Toast
+
+
+class PasswordValidator : InputFilter {
+    override fun filter(
+        source: CharSequence,
+        start: Int,
+        end: Int,
+        dest: Spanned,
+        dstart: Int,
+        dend: Int
+    ): CharSequence? {
+        val password = dest.toString() + source.toString()
```

Comment:
Why do you combine source and dest?

Reply:
Take a look at your teammate's PR and try to use similar approach
https://github.com/SafetyHeads/Akademia-Androida-2023/pull/26/files